k8s/upgrade/src/bin/upgrade-job/upgrade/utils.rs

```diff
@@ -24,6 +25,7 @@ pub(crate) struct RebuildResult {
 pub(crate) async fn rebuild_result(
     rest_client: &RestClientSet,
     stale_volumes: &mut Vec<Volume>,
+    node_name: &str,
```

Comment:
Is kubernetes node-name and mayastor node name always guaranteed to be the same? Are mayastor pools tied to the nodename or the `kubernetes.io/hostname` label value?
cc: @tiagolobocastro @Abhinandan-Purkait